howto/cluster/scale-down.md

```diff
@@ -1,24 +1,35 @@
 Scaling down a LXD cluster involves more checks than scaling up.
 
-First, you must pick a suitable candidate node. When a node is removed, all still running containers on it are stopped and potentially connected users are disconnected. To avoid this, AMS provides a feature to mark a node as unschedulable so that it will not be considered for any further container launches.
+There are two important things to know when scaling down:
+ - The node you remove must not have any container left.
+ - You must wait for a node to be fully removed before you can start removing another one
```

Comment:
```suggestion
There are two important requirements when scaling down:
 - The node you remove must not have any containers left.
 - You must wait for a node to be fully removed before you can start removing another one.
```
There were a few more changes in here. If you prefer your intro that's fine with me, but I'll insist on the plural and the period at the end. ;)